Resources/Prototypes/Entities/Objects/Consumable/Food/soup.yml

```diff
@@ -566,6 +566,31 @@
         - ReagentId: Amatoxin
           Quantity: 6
 
+- type: entity
+  name: scrambled eggs
+  parent: FoodBowlBase
+  id: FoodMealScrambledEggs
+  description: Some cooked eggs with salt. Yummy!
+  components:
+  - type: FlavorProfile
+    flavors:
+      - egg
+      - salty
+  - type: Sprite
+    layers:
+    - state: bowl
+    - state: scrambled-eggs
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxvol: 10
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 9
+        - ReagentId: TableSalt
+          Quantity: 1
```

Comment:
Replace `TableSalt` with `Protein` (and adjust the amount, like 5 of each) as there is no need to add salt to a dish that tells you it's salty. This also means chefs won't be able to get a possible reimbursement on TableSalt.